modules/test/playwright/pages/commerce/commerce-channel-web/commerceAdminChannelDetailsPage.ts

```diff
@@ -327,6 +335,104 @@ export class CommerceAdminChannelDetailsPage {
 		await waitForAlert(await this.sidePanelNestedFrame('Shipping Methods'));
 	}
 
+	async addFixedTaxRate(amount: string, name: string) {
+		const tableName = 'Tax Calculations';
+
+		await (
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).click();
+
+		await expect(
+			(await this.sidePanelFrame(tableName)).getByLabel('Active')
+		).toBeVisible();
+
+		await (await this.sidePanelFrame(tableName))
+			.getByLabel('Active')
+			.check();
+		await (await this.frameSaveButton(false, tableName)).click();
+
+		await (await this.taxRatesTab(tableName)).click();
+		await (await this.sidePanelFrame(tableName))
+			.getByTitle('Add Tax Rate')
+			.click();
+
+		await expect(
+			this.addTaxRateFrame.getByText('Tax Category')
+		).toBeVisible();
+
+		await this.addTaxRateFrame
+			.getByLabel('Tax Category')
+			.selectOption(name);
+		await this.addTaxRateFrame.getByLabel('Amount').fill(amount);
+		await this.addTaxRateFrame
+			.getByRole('button', {name: 'Submit'})
+			.click();
+
+		await this.page.reload();
+
+		await expect(
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).toBeVisible();
+	}
+
+	async editFixedTaxRate(newAmount: string, name: string) {
+		const tableName = 'Tax Calculations';
+
+		await (
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).click();
+		await (await this.taxRatesTab(tableName)).click();
+
+		await (await this.getRowByTextFromSidePanelTable(tableName, name))
+			.getByRole('button', {name: 'Actions'})
+			.click();
+
+		await (await this.sidePanelFrame(tableName))
+			.getByRole('menuitem', {name: 'Edit'})
+			.click();
+
+		await (await this.sidePanelNestedFrame(tableName))
+			.getByLabel('Amount')
+			.fill(newAmount);
+		await (await this.frameSaveButton(true, tableName)).click();
+
+		await this.page.reload();
+
+		await expect(
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).toBeVisible();
+	}
+
+	async getRowByTextFromSidePanelTable(tableName: string, text: string) {
+		return (await this.sidePanelFrame(tableName))
+			.locator('.dnd-tbody')
+			.locator('.dnd-tr')
+			.filter({
+				has: (await this.sidePanelFrame(tableName))
+					.getByText(text, {exact: true})
+					.first(),
+			});
+	}
+
+	async verifyFixedTaxRate(amount: string, name: string) {
+		const tableName = 'Tax Calculations';
+
+		await (
+			await this.generalCommerceAdminChannelTableLink('Fixed Tax Rate')
+		).click();
+		await (await this.taxRatesTab(tableName)).click();
+
+		await expect(
+			(await this.sidePanelFrame(tableName)).getByText(name)
+		).toBeVisible();
+
+		await expect(
+			(
+				await this.getRowByTextFromSidePanelTable(tableName, name)
+			).locator('.cell-rate')
+		).toHaveText(amount);
+	}
```

Comment:
is this correct to be there? I mean... it seems to be more a "test thing" compared that a page "behavior"